tests/console/validate_modify_existing_partition.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify the partition modified in modify_existing_partition.
+# Maintainer: Jonathan Rivrain <jrivrain@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data  = get_test_data();
+    my $fs_type    = $test_data->{fs_type};
+    my $partition  = $test_data->{existing_partition};
+    my $mountpoint = $test_data->{mount_point};
+    my $part_size  = $test_data->{part_size};
+    $part_size =~ s/[A-Za-z]//g;
+
+    select_console "root-console";
+    record_info("Check $fs_type", "Verify that the partition filesystem is $fs_type");
```

Comment:
I would avoid to write variables in the first parameter as it has a small space for visualization, but it is ok, should fit any format probably.

Reply:
Removed